### PR TITLE
tb changes, sfifo make, removed log2, readme

### PR DESCRIPTION
**Pull request by josejsarmento**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,7 @@
-# README #
+# IOb-mem #
 
-This README would normally document whatever steps are necessary to get your application up and running.
+Memory modules used in IOb-SoC. For more information, please check: [IOb-SoC](https://github.com/IObundle/iob-soc)
```
Comment (jjts) [inline README.md:3]: Not for iob-soc only, please remove this sentence . This is a genereic mem lib

File: README.md [hunk 2]
```diff
@@ -1,29 +1,7 @@
...
 
-### What is this repository for? ###
+### Compile a specific memory module ###
```
Comment (jjts) [inline README.md:5]: we're not compiling

we use this repos a subbmodule and grab its components 

we may simulate each to check how to use and if they are OK

Comment (josejsarmento) [reply README.md:5]:
```
# IOb-mem #

### Simulate a specific memory module ###

Go to the memory module folder and type `make`.
```
Is this correct?

**Comment by jjts** [comment]
cannot accept because github won't show the diffs

**Pull request closed without merging**